about.html

```diff
@@ -56,22 +56,25 @@
     </div>
     <div class="introduction">
       <div class="main_heading">
-        <h1>KENYA YOUTH SUMMIT</h1>
+        <h1>KENYA YOUTH SUMMIT 2022</h1>
       </div>
-      <h2><span> Youth Summit></span> , held every two years, is a place where youths from all over 
+      <p class="intro"><span>Kenya Youth Summit</span> , held every two years, is a place where youths from all over 
                Kenya , scholars and activists from all walks of life who support and practice open culture gather together
-      </h2>
-      <p>
-        In particular, this year's CC Summit
-        awaits the participation of various organizations and individual participants who want to participate in 
-        activities to spread the value of sharing, free culture, and open knowledge with CC.
-        Free software supporters, Wikipedia activists, open knowledge foundation activists, art galleries, libraries, museums, archives, governments,
-        foundations, lawyers, social activists, etc. various people active in the field of open content and open knowledge gather in one place to
-        share richer and more dynamic We hope that it will be an opportunity to share our thoughts on creating an ecosystem
       </p>
-      <span class="dates">
-        October 14 - 17, 2015 (Wednesday - Saturday) See you at the CC Global Summit! 
-      </span>
+      <div class="explanation">
+        <p class="details">
+          In particular, this year's CC Summit
+          awaits the participation of various organizations and individual participants who want to participate in 
+          activities to spread the value of sharing, free culture, and open knowledge with CC.
+          Free software supporters, Wikipedia activists, open knowledge foundation activists, art galleries, libraries, museums, archives, governments,
+          foundations, lawyers, social activists, etc. various people active in the field of open content and open knowledge gather in one place to
+          share richer and more dynamic We hope that it will be an opportunity to share our thoughts on creating an ecosystem
+        </p>
+        <span class="dates">
+          October 14 - 17, 2015 (Wednesday - Saturday) See you at the CC Global Summit! 
+        </span>
```

Comment:
- [ ] Kindly center all your elements to the left side as seen in the original design 👍 
- Your design:
![Annotation 2022-06-24 012912](https://user-images.githubusercontent.com/85113567/175439258-11fb97d4-0f29-4a13-9340-fbffbab3d035.jpg)
- Required design:
![Annotation 2022-06-24 012929](https://user-images.githubusercontent.com/85113567/175439276-6c887663-cd99-4363-a2c3-39abeabc41ed.jpg)